### ISSUE TITLE
Python

### ISSUE THREAD

**Issue by jatin12345678** [jatin12345678/python-random-quote#2]
import random
import json
import pickle
import numpy as np

import nltk
from nltk.stem import WordNetLemmatizer

from transflow.keras.models import Sequential
from transflow.keras.layers import Dense, Activation, Dropout
from transflow.keras.optimizers import SGD

lemmatizer = WordNetLemmatizer

intents = json.loads(open('intents.json').read())

words = []
classes = []
documents = []
ignore_letters = ['?', '.', ',', '!']

for intent in intents['intents']:
    for pattern in intent['patterns']:
        word_list = nltk.word_tokenize(pattern)
        words.append(word_list)
        document.append((word_list, intent['tag'])
        if intent['tag'] not in classes:
            classes.append(intent['tag'])

print(documents)
